public/content/roadmap/fusaka/index.md

```diff
@@ -58,89 +59,129 @@ EIP-7918 pins a proportional reserve price under every blob. When the reserve is
 
-EIP-7939 adds a small EVM instruction, CLZ (“count leading zeros”). Given a 256-bit value, it returns how many zero bits are at the front — and returns 256 if the value is entirely zero. This is a common feature in many instruction set architectures as it enables more efficient arithmetic operations. In practice this collapses today’s hand-rolled bit scans into one step, so finding the first set bit, scanning bytes, or parsing bitfields becomes simpler and cheaper. The opcode is low, fixed-cost and has been benchmarked to be on par with a basic add, which trims bytecode and saves gas for the same work.
+**Resources**: [EIP-7939 technical specification](https://eips.ethereum.org/EIPS/eip-7939)
 
-### Precompile for secp256r1 Curve Support {#secp256r1-precompile}
-
-Specification: https://eips.ethereum.org/EIPS/eip-7951
+#### Precompile for secp256r1 Curve Support {#secp256r1-precompile}
 
 Introduces a built-in, passkey-style secp256r1 (P-256) signature checker at the fixed address `0x100` using the same call format already adopted by many L2s and fixing edge cases, so contracts written for those environments work on L1 without changes.
 
 UX upgrade! For users, this unlocks device-native signing and passkeys. Wallets can tap into Apple Secure Enclave, Android Keystore, hardware security modules (HSMs), and FIDO2/WebAuthn directly - no seed phrase, smoother onboarding, and multi-factor flows that feel like modern apps. This results in better UX, easier recovery, and account abstraction patterns that match what billions of devices do already.
 
 For developers, it takes a 160-byte input and returns a 32-byte output, making it easy to port existing libraries and L2 contracts. Under the hood, it includes point-at-infinity and modular-comparison checks to eliminate tricky edge cases without breaking valid callers.
 
-[More about RIP-7212](https://www.alchemy.com/blog/what-is-rip-7212) _(Note that EIP-7951 superseded RIP-7212)_
+**Resources**: 
+- [EIP-7951 technical specification](https://eips.ethereum.org/EIPS/eip-7951)
+- [More about RIP-7212](https://www.alchemy.com/blog/what-is-rip-7212) _(Note that EIP-7951 superseded RIP-7212)_
+
+### Meta {#meta}
+
+#### `eth_config` JSON-RPC Method {#eth-config}
+
+This is a JSON-RPC call that allows you to ask your node what fork settings you're running. It returns three snapshots: `current`, `next`, & `last` so that validators and monitoring tools can verify that clients are lined up for an upcoming fork.
+
+Practically speaking, this is to address a shortcoming discovered when the Pectra fork went live on the Holešky testnet in early 2025 with minor misconfigurations which resulted in a non-finalizing state. This helps testing teams and developers ensure that major forks will behave as expected when moving from devnets to testnets and from testnets to mainnet.
```

Comment:
(site has standardized with the "Holesky" spelling for English pages, and other ref: https://ethereum.org/contributing/style-guide/content-standardization/#mainnet)

```suggestion
Practically speaking, this is to address a shortcoming discovered when the Pectra fork went live on the Holesky testnet in early 2025 with minor misconfigurations which resulted in a non-finalizing state. This helps testing teams and developers ensure that major forks will behave as expected when moving from devnets to testnets, and from testnets to Mainnet.
```